### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by talrejah02**
please review and give suggestions

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,12 +1,148 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (haarsh24) [inline index.html:7]: Suggestion:
Add author and description for better SEO

File: index.html [hunk 2]
```diff
@@ -1,12 +1,148 @@
...
+        </nav>
+        
+        <section class="hero">
+            <img class="hero-image"
+                src="https://www.decathlon.in/_next/image?url=https%3A%2F%2Fcontents.mediadecathlon.com%2Fs896860%2Fk%24382f8943451fc70c7f2ae3d4ab3db60a%2Fdesktop.jpg%3Ff%3D0x250%3Ff%3D1920x470%26format%3Dauto&w=1920&q=75" />
+        </section>
+
+        <section class="category-container">
+            <section class="category-card">
+                <img class="category-card-img" src="./Assets/images/jackets.jpg" />
```
Comment (haarsh24) [inline index.html:75]: Add alt in img tags

File: style.css [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+    margin: 0;
+    letter-spacing: 1.5px;
+    box-sizing: border-box;
+}
+
+.nav {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    align-items: center;
```
Comment (haarsh24) [inline style.css:30]: You can declare these properties in util and reuse this as class in multiple areas

File: style.css [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+    z-index: 1;
+
+}
+
+
+.header-logo {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
```
Comment (haarsh24) [inline style.css:43]: Same here

File: style.css [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+
+.header-svg {
+    display: flex;
+    justify-content: center;
+}
+
+.header-search {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
```
Comment (haarsh24) [inline style.css:63]: Here too

File: style.css [hunk 6]
```diff
@@ -0,0 +1,209 @@
...
+
+.header-search-input:focus {
+    outline: none;
+}
+
+.nav-right {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    align-items: center;
```
Comment (haarsh24) [inline style.css:87]: same

File: pages/product/product.css [hunk 7]
```diff
@@ -0,0 +1,201 @@
...
+.main-wrapper {
+    display: flex;
+    width: 100%;
+    height: 100%;
+}
+
+.filter-wrapper {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: column;
```
Comment (haarsh24) [inline pages/product/product.css:12]: same

File: pages/product/product.css [hunk 8]
```diff
@@ -0,0 +1,201 @@
...
+
+/* -----------------card--------------- */
+.card {
+    position: relative;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    flex-wrap: wrap;
+    padding: 0rem 0.5rem 0.2rem 0.5rem;
+    background-color: var(--white);
```
Comment (haarsh24) [inline pages/product/product.css:90]: You can use better name than white

File: pages/product/product.css [hunk 9]
```diff
@@ -0,0 +1,201 @@
...
+    color: var(--secondary);
+}
+
+.product-price {
+    font-size: 0.8rem;
+    padding-bottom: 0.5rem;
+}
+
+.price {
+    background-color: var(--accent);
```
Comment (haarsh24) [inline pages/product/product.css:130]: Use good name ,

**Review by haarsh24 — CHANGES_REQUESTED** [review]
Suggested some changes , after that you can merge ,

File: pages/product/product.css [hunk 10]
```diff
@@ -0,0 +1,201 @@
...
+    border-top-right-radius: 3px;
+    border-bottom-right-radius: 3px;
+    top: 5px;
+    left: 0px;
+}
+
+.two {
+    background-color: var(--accent);
+    color: var(--secondary);
+    top: 22px;
```
Comment (Jia-A) [inline pages/product/product.css:184]: You can change this px to rem.

**Review by ShreyashMahajan — APPROVED** [review]
Good to merge. Just look at suggestions which harsh has suggested

File: style.css [hunk 11]
```diff
@@ -0,0 +1,209 @@
...
+
+.category-card-subtext {
+    font-weight: 600;
+    font-size: 2.1rem;
+}
+
+.category-card-btn {
+    position: absolute;
+    padding: 0.8rem 2rem;
+    bottom: 20px;
```
Comment (Jia-A) [inline style.css:167]: Again px to rem can be changed.

File: pages/product/product.css [hunk 12]
```diff
@@ -0,0 +1,201 @@
...
+}
+
+/* -----------------card--------------- */
+.card {
+    position: relative;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    flex-wrap: wrap;
+    padding: 0rem 0.5rem 0.2rem 0.5rem;
```
Comment (SaurabhChawda) [inline pages/product/product.css:89]: Use Shorthand Property.

File: pages/product/product.css [hunk 13]
```diff
@@ -0,0 +1,201 @@
...
+    background-color: var(--white);
+    max-width: 225px;
```
Comment (SaurabhChawda) [inline pages/product/product.css:91]: Use Rem as much as possible.

File: pages/product/product.css [hunk 14]
```diff
@@ -0,0 +1,201 @@
...
+    display: flex;
+    flex-direction: column;
+}
+
+.card-detail {
+    display: flex;
+    flex-direction: column;
+    font-size: 1rem;
+    padding-top: 0.5rem;
+    padding-bottom: 0.5rem;
```
Comment (SaurabhChawda) [inline pages/product/product.css:112]: Here also.

**Review by SaurabhChawda — APPROVED** [review]

**Review by Jia-A — COMMENTED** [review]
One thing you can do is make a css file with different flex properties in different classes and then use them everywhere, as you've used them multiple times, it will be beneficial.

**Review by Jia-A — APPROVED** [review]

**Pull request merged**